Error when running "cypress open"

### Current behavior

When I run "cypress open", I expect the cypress app to open, but it fails and throws an error as shown in the following figure.

<img width="1045" height="507" alt="Image" src="https://github.com/user-attachments/assets/93e70c0e-c261-4fcc-bae9-b7c394bbf26f" />

### Debug logs

```Text
PS F:\cypress project> npm run cy:open

> cy:open
> cypress open



DevTools listening on ws://127.0.0.1:63073/devtools/browser/cb954276-77c6-4669-b02e-b94c42dc9d03
<<<CYPRESS.STDERR.START>>>node:internal/modules/esm/resolve:265
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'C:\Users\IT' imported from F:\cypress project\
    at finalizeResolution (node:internal/modules/esm/resolve:265:11)
    at moduleResolve (node:internal/modules/esm/resolve:933:10)
    at defaultResolve (node:internal/modules/esm/resolve:1157:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:390:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:359:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:234:38)
    at ModuleLoader.import (node:internal/modules/esm/loader:322:34)
    at loadESM (node:internal/process/esm_loader:23:33)
    at runMainESM (node:internal/modules/run_main:98:21)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:131:5) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///C:/Users/IT'
}

Node.js v20.12.2
<<<CYPRESS.STDERR.END>>>
```

### Cypress Version

15.0.0

### Node version

20.12.2

### Package Manager

npm

### Package Manager Version

10.8.0

### Operating system

Windows

### Operating System Version

10

### Other

Cypress v14.2.0 works fine with me.

My windows account user name is "IT SHOP", which makes the issue, as Cypress is trying to get something from a path containing "IT SHOP", but it fails to read it as "IT SHOP", it fails to read the space in the middle and just read "IT".

One solution is to change my user name, but my user name works fine on Cypress v14.2.0 and shouldn't be an issue, so this issue has been introduced in a newer version.

Maybe Cypress concatenates the account user name into a path to use it in the terminal without putting the whole path into string quotes.

--- a/.circleci/workflows.yml
+++ b/.circleci/workflows.yml
@@ -83,7 +83,7 @@ windowsWorkflowFilters: &windows-workflow-filters
       - equal: [ develop, << pipeline.git.branch >> ]
       # use the following branch as well to ensure that v8 snapshot cache updates are fully tested
       - equal: [ 'update-v8-snapshot-cache-on-develop', << pipeline.git.branch >> ]
-      - equal: [ 'fix/set_fixed_gtk_version', << pipeline.git.branch >> ]
+      - equal: [ 'bugfix/tsx_loader_path', << pipeline.git.branch >> ]
       - matches:
           pattern: /^release\/\d+\.\d+\.\d+$/
           value: << pipeline.git.branch >>
@@ -157,7 +157,7 @@ commands:
           name: Set environment variable to determine whether or not to persist artifacts
           command: |
             echo "Setting SHOULD_PERSIST_ARTIFACTS variable"
-            echo 'if ! [[ "$CIRCLE_BRANCH" != "develop" && "$CIRCLE_BRANCH" != "release/"* && "$CIRCLE_BRANCH" != "fix/set_fixed_gtk_version" ]]; then
+            echo 'if ! [[ "$CIRCLE_BRANCH" != "develop" && "$CIRCLE_BRANCH" != "release/"* && "$CIRCLE_BRANCH" != "bugfix/tsx_loader_path" ]]; then
                 export SHOULD_PERSIST_ARTIFACTS=true
             fi' >> "$BASH_ENV"
   # You must run `setup_should_persist_artifacts` command and be using bash before running this command

--- a/cli/CHANGELOG.md
+++ b/cli/CHANGELOG.md
@@ -13,6 +13,7 @@ _Released 08/26/2025 (PENDING)_
 - Fixed an issue where Cypress would fail to run on GNOME if GTK 4 and GTK 2/3 were detected in the Electron process. Addresses [#32361](https://github.com/cypress-io/cypress/issues/32361).
 - Fixed an issue where the open Studio button would incorrectly show for component tests. Addressed in [#32315](https://github.com/cypress-io/cypress/pull/32315).
 - Fixed an issue where the TypeScript compiler wasn't being resolved correctly when `@cypress/webpack-batteries-included-preprocessor` was used as a standalone package. Fixes [#32338](https://github.com/cypress-io/cypress/issues/32338).
+- Fixed an issue where `tsx` was not being loaded correctly into the Cypress configuration process due to spaces being present in the path. Fixes [#32398](https://github.com/cypress-io/cypress/issues/32398).
 
 **Misc:**
 

--- a/packages/data-context/src/data/ProjectConfigIpc.ts
+++ b/packages/data-context/src/data/ProjectConfigIpc.ts
@@ -312,10 +312,11 @@ export class ProjectConfigIpc extends EventEmitter {
     /**
      * use --import for node versions
      * 20.6.0 and above for 20.x.x as --import is supported
-     * use --loader for node under 20.6.0 for 20.x.x
+     * use --loader for node under 20.6.0 for 20.x.x.
+     * NOTE: we need to use double quotes around the tsx path to account for paths with spaces or special characters.
      * @see https://tsx.is/dev-api/node-cli#node-js-cli
      */
-    let tsxLoader = this.nodeVersion && semver.lt(this.nodeVersion, '20.6.0') ? `--loader ${tsx}` : `--import ${tsx}`
+    let tsxLoader = this.nodeVersion && semver.lt(this.nodeVersion, '20.6.0') ? `--loader "${tsx}"` : `--import "${tsx}"`
 
     // If they've got TypeScript installed, we can use tsx for CommonJS and ESM.
     // @see https://tsx.is/dev-api/node-cli#node-js-cli

--- a/packages/data-context/test/unit/data/ProjectConfigIpc.spec.ts
+++ b/packages/data-context/test/unit/data/ProjectConfigIpc.spec.ts
@@ -85,14 +85,14 @@ describe('ProjectConfigIpc', () => {
                 // For node 20.5.1 and down, we need use the --loader flag
                 expect(forkSpy).to.have.been.calledWith(sinon.match.string, sinon.match.array, sinon.match({
                   env: {
-                    NODE_OPTIONS: sinon.match(/--loader .*cypress\/node_modules\/tsx\/dist\/loader.mjs/),
+                    NODE_OPTIONS: sinon.match(/--loader ".*cypress\/node_modules\/tsx\/dist\/loader.mjs"/),
                   },
                 }))
               } else {
                 // For node 20.6.0 and up, we need use the --import flag
                 expect(forkSpy).to.have.been.calledWith(sinon.match.string, sinon.match.array, sinon.match({
                   env: {
-                    NODE_OPTIONS: sinon.match(/--import .*cypress\/node_modules\/tsx\/dist\/loader.mjs/),
+                    NODE_OPTIONS: sinon.match(/--import ".*cypress\/node_modules\/tsx\/dist\/loader.mjs"/),
                   },
                 }))
               }

--- a/packages/stderr-filtering/package.json
+++ b/packages/stderr-filtering/package.json
@@ -5,7 +5,7 @@
   "main": "dist/index.js",
   "scripts": {
     "build": "yarn clean && tsc",
-    "clean": "rm -rf ./{dist,coverage}",
+    "clean": "rimraf -rf ./{dist,coverage}",
     "lint": "eslint",
     "test": "vitest",
     "watch": "tsc --watch"
@@ -18,6 +18,7 @@
     "@types/node": "^22.13.10",
     "@vitest/coverage-v8": "^3.0.9",
     "eslint": "^9.22.0",
+    "rimraf": "^6.0.1",
     "typescript": "^5.8.2",
     "vitest": "^3.0.9"
   },

